source/hardware/controllers/numark_mixtrack_pro.rst

```diff
@@ -28,65 +28,89 @@ Mapping Description
-16. **PLAY / PAUSE:** Starts or resumes playback if the Deck is paused. Pauses playback if the Deck is playing. LED ON when playing. LED OFF when stopped.
-17. **STUTTER:** Adjust BeatGrid in the correct place (useful to sync well).
-    **\*LED Blink at each Beat**\ *of the grid.*
-18. **CUE:** The CUE button will return and pause the track at the last set cue point. For temporary play of the cue point, you can hold down the CUE button. The track will play for as long as the
-    button is held down and will return to the cue point once it has been released (Until it has been pressed PLAY before release CUE, then playback will continue). LED ON when stopped. LED OFF when
-    playing.
-    **\*LED Blink at each Beat in the last 30 seconds of song.
-    \*Don’t set Cue accidentally at the end of the song (return to the latest cue).**
-19. **SYNC:** Automatically matches the corresponding Deck’s tempo and phase with the other Deck.
-    **\*If the other deck is stopped, only sync tempo (not phase).
-    \*LED Blink at Clip Gain**\ *(Peak indicator).
-    *\ **\*Pressing DELETE**\ *before SYNC: Set Pitch to 0%.*
+14. **JOG WHEEL:** When the deck is playing, the Jog Wheel will bend the track’s pitch. When the Scratch button is on, touching the top of the Jog Wheel will scratch the selected track **(can do
+    backspin effect)**, and touching the border of the Jog Wheel will bend the track’s pitch. When the deck is not playing, rotating the Jog Wheel will scan through the track.
+15. **SCRATCH:** Turns Scratch Mode on or off. If Scratch Mode is on, the button will light up and the Jog Wheel will scratch like a turntable when you spin it.
+16. **PLAY/PAUSE:** Starts or resumes playback if the Deck is paused. Pauses playback if the Deck is playing. LED is on when playing and off when stopped.
+
+    * **Delete + PLAY/PAUSE:** Start/stop playback with a vinyl speed-up/slow-down effect
+
+17. **STUTTER:** Adjust BeatGrid in the correct place (useful to sync well). The Stutter button will blink once per beat.
+18. **CUE:** Sets and navigates the track to a Cue point. Behavior emulates popular CDJs and depends on whether the relevant deck is currently playing:
+
+    * While paused:
+
+        * Press and release the Cue button to set the Cue point.
+        * Press and hold the Cue button to temporarily play the track from the last Cue point. When the button is released, playback will stop; or, while holding the Cue button, press the Play button to continue playback.
+
+    * While playing:
+
+        * Press and release the Cue button to return the track to the last set Cue point and pause.
+
+    * *Note:* The Cue button will blink once per beat during the last 30 seconds of the song.
+    * *Note:* If Cue is pressed at the end of the song, no cue will be set; instead, you will return to the most recent cue.
```

Comment:
Doesn't this depend on the cue behavior configured in the preferences? If not, that should be changed IMHO.